package.json

```diff
@@ -29,7 +28,6 @@
     "leaflet-distortableimage": "^0.6.0",
     "leaflet-draw": "Leaflet/Leaflet.draw#0.2.4",
     "leaflet-easybutton": "CliffCloud/Leaflet.EasyButton#1.0.0",
-    "leaflet-environmental-layers": "^1.3.1",
```

Comment:
Attempt to install this dependency gives the following output:

![image](https://user-images.githubusercontent.com/19597045/60685630-7d7c2f00-9e7a-11e9-9a52-96fda6d08272.png)